java/arcs/jvm/host/ExplicitHostRegistry.kt

```diff
@@ -10,22 +10,31 @@
  */
 package arcs.jvm.host
 
+import arcs.core.host.ArcHost
+import arcs.core.host.HostRegistry
 import arcs.sdk.Particle
-import kotlin.reflect.KClass
 
 /**
  * A []HostRegistry] that discovers the available [ArcHost]s available on this platform by using
  * explicitly registered [ArcHost]s and [Particle]s invoked by [HostRegistry.registerHost] and
  * [ExplicitHostRegistry.registerParticles].
  */
-object ExplicitHostRegistry : AnnotationBasedHostRegistry() {
+object ExplicitHostRegistry : HostRegistry {
+    private val arcHosts = mutableListOf<ArcHost>()
```

Comment:
It feels rather dangerous to be maintaining static/global bindings to potentially-very-heavy objects?

Reply:
Changed to a class instance.